1-JavaScript/Week2/assignment/ex1-giveCompliment.js

```diff
@@ -1,36 +1,36 @@
 'use strict';
-/* -----------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-JavaScript/Week3#exercise-1-you-are-amazing
-
-1. Complete the function named `giveCompliment`as follows:
-
-   - It should take a single parameter: `name`.
-   - Its function body should include a variable that holds an array,
-     `compliments`, initialized with 10 strings. Each string should be a
-     compliment, like `"great"`, `"awesome"` and so on.
-   - It should randomly select a compliment from the array.
-   - It should return the string "You are `compliment`, `name`!", where
-     `compliment` is a randomly selected compliment and `name` is the name that
-     was passed as an argument to the function.
-
-2. Call the function three times, giving each function call the same argument:
-   your name.
-   Use `console.log` each time to display the return value of the
-   `giveCompliment` function to the console.
------------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+
+function giveCompliment(name) {
+  
+  const compliments = [
```

Comment:
Maybe just a personal styling note: I would remove the spaces between the function definition and the body of the function. I have also noticed some places where there's multiple blank lines, I would reduce that to 1.

```suggestion
function giveCompliment(name) {
  const compliments = [
```